plugins/modules/gpg_secretstore.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2021-2022, Famedly GmbH
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: gpg_secretstore
+author:
+    - Jadyn Emma Jäger <j.jaeger@famedly.com> (@_)
```

Comment:
Please update accordingly

Reply:
Any preference for what you'd like to have in there? I can rewrite them for the past as well.

Reply:
I've gone for `Jadyn Emma Jäger (@jadyndev)` for now, if you want that changed please update it in a follow-up PR.